pkg/controller/config/match/match.go

```diff
@@ -0,0 +1,77 @@
+package match
+
+import (
+	"sync"
+
+	configv1alpha1 "github.com/open-policy-agent/gatekeeper/apis/config/v1alpha1"
+)
+
+type Operation string
+
+const (
+	Audit   = Operation("audit")
+	Sync    = Operation("sync")
+	Webhook = Operation("webhook")
+	Star    = Operation("*")
+)
+
+type Set struct {
+	mux                sync.RWMutex
+	ExcludedNamespaces map[Operation]map[string]bool
```

Comment:
This should not be publicly accessible due to thread safety issues.

Reply:
added, `GetExcludedNamespaces()` to retrieve this